### PR TITLE
Fixed Issue with .listToArray and .each, not supported in CF11

### PR DESCRIPTION
**Pull request by LucidSolutions**
Fixed Issue with .listToArray and .each, not supported in CF11 @lmajano @gpickin

### CODE REVIEW

File: modules/contentbox/models/modules/ModuleService.cfc [hunk 1]
```diff
@@ -378,24 +378,24 @@ component extends="cborm.models.VirtualEntityService" accessors="true" singleton
 		};
 
 		// Register Core
-		qCoreModules.each( function( row ){
+		for(row in qCoreModules){
```
Comment (lmajano) [inline modules/contentbox/models/modules/ModuleService.cfc]: Any reason why this change?

Comment (LucidSolutions) [reply modules/contentbox/models/modules/ModuleService.cfc]: Luis ... we have done this because of Each loop on query object is not supported in CF11

Pleae see this error we get.

cbissue

**Comment by GunnarLieb** [comment]
row is not var scoped, or am I missing something?

**Comment by LucidSolutions** [comment]
@GunnarLieb Yes, we need to do that. Thanks for pointing that out.

File: modules/contentbox/models/modules/ModuleService.cfc (hunk 1 above)
Comment (lmajano) [reply modules/contentbox/models/modules/ModuleService.cfc]: Arggghhh good old Adobe
